packages/EasyErrorHandler/src/Builders/ValidationErrorResponseBuilder.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Builders;
+
+use ApiPlatform\Core\Bridge\Symfony\Validator\Exception\ValidationException;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
```

Comment:
We can't do this or it means this package won't work on application that don't have api-platform, need to revisit the solution.

Reply:
Removed

Reply:
@alexandrtspl we can add `EonX\EasyErrorHandler\Bridge\ApiPlatform\Builder\ValidationErrorResponseBuilder` that will be specific to API Platform.

Reply:
Code added to Bridge/Symfony/Builder/ApiPlatformValidationExceptionBuilder.php

Reply:
@natepage as soon as `Bridge/Symfony/Builder/ApiPlatformValidationExceptionBuilder.php` is specific to API Platform and is enabled only when the specific class exists, I think it should be fine to go this way.

Reply:
Yes I think so, thanks